pyoptsparse/pyOpt_utils.py

```diff
@@ -1,16 +1,14 @@
 #!/usr/bin/env python
 """
-pyOptSparse_utils
-
-Holds a minimal set of sparse-matrix type routines for pyOptSparse. This
+pyOptSparse_utils holds a minimal set of sparse-matrix type routines for pyOptSparse. This
 is designed to replace the HORRENDOUS scipy sparse matrix format. The
 with scipy.sparse is that is the NO way to enforce a constant sparsity
```

Comment:
I'm just pointing out typos since I'm reading this anyway. It should say 'The problem with scipy.sparse is that there is...'